modules/internal/ChapelRange.chpl

```diff
@@ -1075,8 +1084,8 @@ operator :(r: range(?), type t: range(?)) {
   }
 
   /*
-     If ``ind`` is a member of the range's represented sequence, returns
-     an integer giving the ordinal index of ind within the sequence
+     If ``ind`` is a member of the range's represented sequence, this returns
```

Comment:
The phrasing of "this returns" seems unique here. I would choose to reword this to something along the lines of: "Returns an integer representing the ordinal index of ``ind`` within the sequence using zero-based indexing if ``ind`` is a member of the range's represented sequence. Otherwise, returns -1."

This makes it more consistent with the other docs and puts the `if`/`otherwise` closer together to make it easier to keep track of, but this is minor and I don't think needs to change, regardless of whatever the length of this message implies.

Reply:
Done, thanks!